Update User Guide - Counter Widget options and 3D models options (#10306)
geosolutions-it/MapStore2: docs/user-guide/widgets.md

!!! Note
    More information about the syntax options allowed for **Format** are available [here](https://d3-wiki.readthedocs.io/zh_CN/master/Formatting/) and the allowed expression to be used as **Formula** are available [here](https://github.com/m93a/filtrex#expressions) in the online documentation.

> * Choose the *Color* for the counter texts through the color picker

Reviewer: ```suggestion
* Choose the *Color* of the Counter text using the color picker
```

> * Select the *Font family* (`Inherit`, `Arial`, `Georgia`, `Impact`, `Tahoma`, `Times New Roman` or `Verdana`) for the counter texts

Reviewer: ```suggestion
* Select the *Font family* (`Inherit`, `Arial`, `Georgia`, `Impact`, `Tahoma`, `Times New Roman` or `Verdana`) of the Counter text
```
